trade-finance-manager-ui/server/controllers/case/amendments/index.js

```diff
@@ -0,0 +1,88 @@
+const {
+  getUnixTime,
+} = require('date-fns');
+
+const api = require('../../../api');
+const { amendmentRequestDateValidation } = require('./requestDateValidation');
+
+const CONSTANTS = require('../../../constants');
+
+// when add an amendment button clicked, renders amendment request date page
+const getAmendmentRequest = async (req, res) => {
```

Comment:
Try-catch block please.

Reply:
where?

Reply:
In `getAmendmentRequest` as there is an api call, in case it fails we should throw an error message.